src/js/tabs/vtx.js

```diff
@@ -175,6 +179,48 @@ TABS.vtx.initialize = function (callback) {
         vtxcallback_after_read();
     }
 
+    function getSmartAudioVersion()
+    {
+        const sa = FC.SMARTAUDIO_DEVICE.version * 100 + FC.SMARTAUDIO_DEVICE.mode;
+        let result = "";
+
+        switch (sa) {
+            case 100:
+                result = "1.0";
+                break;
+            case 116:
+                result = "1.0 unlocked";
```

Comment:
Apart from the review, I think this `unlocked` (and the final unknown) maybe must come from the messages file, to let translate it. I think is not a "standard" word defined in the version. But is only my thought.

Reply:
it's actually a good point, yes!